blogs/ecs-cross-account/account_backend/redis_and_database.yml

```diff
@@ -183,7 +183,7 @@ Resources:
               awslogs-region: !Ref AWS::Region
               awslogs-stream-prefix: "database-envoy"
           Environment:
-            - Name: "APPMESH_VIRTUAL_NODE_NAME"
+            - Name: "APPMESH_RESOURCE_ARN"
```

Comment:
So this new env variable doesn't actually need the full arn?

Reply:
Yes, that's true.